docs/tutorials/building-your-first-bot.mdx

```diff
@@ -38,11 +38,11 @@ you can't find that version from Unity Hub). Once the project is loaded, open th
 
 ## Import the Regression Games Unity Bots package
 
-You can find [our package on GitHub](https://github.com/Regression-Games/RGUnityBots). Import the package into your Unity project by opening 
+You can find [our package on GitHub](https://github.com/Regression-Games/RGUnityBots). Import the package into your Unity project by opening
 the **Package Manager** window (**Window** > **Package Manager**) and click **Add package from git URL**. Then, paste the following URL:
 
 ```
-https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.20
+https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.25
```

Comment:
Should we add a section after this about our extension packages ?

Something like...

If your game utilizes Unity's ECS, we now have an extension package available to support it.

```
https://github.com/Regression-Games/RGUnityBots/tree/main/src/gg.regression.unity.bots.ecs#v0.0.25
```

Install this package in addition to / after installing the core RGUnityBots package.k

Reply:
Missed this... adding this in 0.0.26 👍